include/dlaf/matrix/tile.h

```diff
@@ -202,6 +319,105 @@ auto create_data(const Tile<T, device>& tile) {
   return common::DataDescriptor<T>(tile.ptr({0, 0}), tile.size().cols(), tile.size().rows(), tile.ld());
 }
 
+/// Create SubTiles:
+/// TODO
+template <class T, Device D>
+std::vector<hpx::shared_future<Tile<const T, D>>> splitTile(
+    hpx::shared_future<Tile<const T, D>>& tile,
+    std::vector<std::tuple<TileElementIndex, TileElementSize>> specs) {
+  std::vector<hpx::shared_future<Tile<const T, D>>> ret;
+  ret.reserve(specs.size());
+  for (const auto& spec : specs) {
+    ret.emplace_back(hpx::dataflow(
+        hpx::launch::sync,
+        [](auto tile, auto origin, auto size) { return Tile<const T, D>(tile, origin, size); }, tile,
+        std::get<0>(spec), std::get<1>(spec)));
+  }
+
+  return ret;
+}
+
+/// Create SubTiles:
+/// TODO
+template <class T, Device D>
+std::vector<hpx::future<Tile<T, D>>> splitTile(
```

Comment:
Would this function be better called `splitTileDisjoint` or something like that? It has stronger preconditions than the read-only version.